mri_probe_ima/mri_probe_ima.cpp

```diff
@@ -549,7 +550,7 @@ static void check_options(void) {
   return;
 }
 /*-------------------------------------------------------------*/
-static int stringmatch(char *s1, char *s2) {
+static int stringmatch(const char *s1, const char *s2) {
```

Comment:
This method appears to be duplicated in a lot of files (there are variants which also get duplicated). I didn't refactor them because, if `char *` is replaced by `std::string` then the equality operator can be used instead (and all of these removed).